content/references/rippled-api/api-conventions/basic-data-types.md

```diff
@@ -0,0 +1,128 @@
+# Basic Data Types
+
+Different types of objects are uniquely identified in different ways:
+
+[Accounts](concept-accounts.html) are identified by their [Address][], for example `"r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59"`. Addresses always start with "r". Many `rippled` methods also accept a hexadecimal representation.
+
+[Transactions](reference-transaction-format.html) are identified by a [Hash][] of the transaction's binary format. You can also identify a transaction by its sending account and [Sequence Number][].
+
+Each closed [Ledger](reference-ledger-format.html) has a [Ledger Index][] and a [Hash][] value. When [Specifying a Ledger Instance](#specifying-ledgers) you can use either one.
+
+## Addresses
+[Address]: #addresses
+
+{% include '_snippets/data_types/address.md' %}
```

Comment:
I am really starting to regret going with this convention of starting folder names with underscores for things that aren't included in the docs directly... 🙀